build-vm

```diff
@@ -566,6 +566,14 @@ vm_detect_2nd_stage() {
 	    echo "resolved swap device $VM_SWAP to $i"
 	    VM_SWAP=$i
 	fi
+        # for zvm, there is just a 4-digig hex number
+            if (( 16#$VM_SWAP )); then
+                if test ${#VM_SWAP} -eq 4; then
+                    sleep 1
+                    VM_SWAP="/dev/$(ls /sys/bus/ccw/devices/0.0.${VM_SWAP}/block/)1"
```

Comment:
why is the ls command needed here at all?

Reply:
Unfortunately, the build worker skript is not clean with regards to VM_SWAP and VM_SWAPDEV. Therefore, I just set it at that point to the real device name as found in /sys.